src/helpers/strings.js

```diff
@@ -120,5 +119,125 @@ export default {
         });
 
         return (params.length > 0) ? _vsprintf(format, params) : format;
+    },
+
+    /**
+     * Changes the string to lowercase.
+     * Example usage:
+     * 		{{lowercase 'JUST WOW!!!'}}   => 'just wow!!!'
+     *
+     * @param  string param
+     * @return string
+     */
+    lowercase: (param) => {
+        if (typeof param === 'undefined' || param === null) {
+            return '';
+        }
+
+        if (typeof param === 'string') {
+            return param.toLowerCase();
+        }
+
```

Comment:
Don't need all these, just this would be fine: 

```javascript
lowercase: (param) => {
    return isString(param) ? param.toLowerCase() : param;
}
```